### ISSUE TITLE
Connection terminated with error: -2146528248

### ISSUE THREAD

**Issue by Rubiloy** [GaryOderNichts/moonlight-wiiu#45]
Trying out the new 1.6 version, once I pair the wiiu moonlight client and then try to connect to my PC, it displays this error.

**Comment by Rubiloy** [comment]
I'm closing this issue since it was because of my PC, not this software. Sorry for the inconveniences.

**Comment by Rubiloy** [comment]
I'm closing this issue since it was because of my PC, not this software. Sorry for the inconveniences.

**Comment by GaryOderNichts** [comment]
Glad you were able to figure it out.
What was the issue on the PC side? That is still an interesting error code.

**Comment by Rubiloy** [comment]
It could have been because of the new Nvidia Game Ready Drivers. At first, I changed the default app to "STREAM" (that's how i called the mstsc.exe app so I could directly stream my desktop) to load from GFE in the Moonlight config file. But once I opened moonlight, paired it with the PC and tried to establish a connection, that error showed up. I later tried changing it back to load Steam instead, but it wouldn't connect either, although this time it just said "Connection terminated".

I jumped too soon into conclusions and came here because of that quite specific error code. But I just downgraded to Nvidia drivers from January and everything is working correctly now! I've just tried to replicate the error, but it ain't doing it.
